src/Services/DatabaseTools/MongoDBDatabaseTool.php

```diff
@@ -77,12 +77,10 @@ public function loadFixtures(array $classNames = [], bool $append = false): Abst
         if ($backupService) {
             $event = new ReferenceSaveEvent($this->om, $executor, $backupService->getBackupFilePath());
             $this->eventDispatcher->dispatch($event, LiipTestFixturesEvents::PRE_REFERENCE_SAVE);
-            $this->testCase->preReferenceSave($this->om, $executor, $backupService->getBackupFilePath());
```

Comment:
is this already documented in the upgrading guide?

Reply:
Yes but not in details: https://github.com/liip/LiipTestFixturesBundle/blob/0006751a3f32ce0b56af9638d0d17304416a02c2/CHANGELOG.md#2x0-tba

Reply:
Done in #105